### PR TITLE
feat(performance): Onboarding page for cache and queues module

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Adds an onboarding page for the Cache and Queues module.
The onboarding page is triggered when a 90 day request for span metrics returns no data.

image

### CODE REVIEW

File: static/app/views/performance/cache/cacheLandingPage.tsx [hunk 1]
```diff
@@ -207,7 +217,7 @@ function PageWithProviders() {
     <ModulePageProviders
       title={[t('Performance'), MODULE_TITLE].join(' — ')}
       baseURL={`/performance/${BASE_URL}`}
-      features="performance-cache-view"
+      features=""
```
Comment (edwardgou-sentry) [inline static/app/views/performance/cache/cacheLandingPage.tsx:220]:
```suggestion
      performance-cache-view
```

File: static/app/views/performance/onboarding/useHasData.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import type {PageFilters} from 'sentry/types';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {useSpanMetrics} from 'sentry/views/starfish/queries/useDiscover';
+
+export function useHasData(filter: string, referrer: string) {
```
Comment (DominikB2014) [inline static/app/views/performance/onboarding/useHasData.tsx:6]: For consistency, i like the idea of `filter` being of type `MutableSearch` to match the api of `useSpanMetrics` better.

File: static/app/views/performance/queues/onboardingContent.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      <Header>{t('Start collecting Insights about your Queues!')}</Header>
+      <p>
+        {t('Our robot is waiting for your first background job to complete.')}{' '}
+        <ExternalLink href="https://develop.sentry.dev/sdk/performance/modules/queues/">
+          {t('Learn more')}
+        </ExternalLink>
+      </p>
+    </Fragment>
+  );
+}
```
Comment (DominikB2014) [inline static/app/views/performance/queues/onboardingContent.tsx:20]: 🤔 The content here, is essentially the same format as queues, and i suspect future modules will be the same, should we just create a single shared component that takes in `header`, `info`, `docsLink`?

**Review by DominikB2014 — APPROVED** [review]
Lgtm, left some nits

**Pull request merged**